jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,15 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+jupyterlab_server==2.3.0
+notebook==6.4.1
+nbdime==3.1.0
+
+# JupyterLab extensions
+jupyterlab-git==0.30.1
+jupyter-contrib-nbextensions==0.5.1
```

Comment:
We can remove this extension because it's [not compatible with jupyterlab](https://github.com/ipython-contrib/jupyter_contrib_nbextensions#jupyterlab).

Reply:
Are we sure about the `jupyter-contrib-nbextensions`? It exists inside the current [minimal]( podman run --rm quay.io/modh/odh-minimal-notebook-container:v1.17.0-9 pip show jupyter-contrib-nbextensions) image that is used by rhods,  Guilliam also  [propose](https://github.com/atheo89/custom-notebooks/blob/main/notebook-controller-images/s2i-minimal-notebook/py38/minimal/Pipfile#L13) it in his notebook

Reply:
@guimou Could you help us clarifying this?

Reply:
Tbh I think it's my repo because it was originally there, as we had some images using Jupyter Classic notebook as the main interface, and there was no direct reason to remove it (like everything else that was linked specifically to JupyterHub).
As those extensions are not compatible with JupyterLab that is now our only target environment, I think it's fine to remove them.
It would be worth to have a look though at other extensions we could bring in though to cover the "migration" from one to the other (autopep8, code-formatter,...): https://jupyterlab-contrib.github.io/migrate_from_classical.html

Reply:
Thank you Guillaume for helping us with this, so, as this extension is not compatible with the jupyter lab, I am going to remove it from the requirements file.